Makefile

```
@@ -116,13 +119,12 @@ else
 endif
 
 CPP        := cpp
-ELF2ROM    := tools/buildtools/elf2rom
 MKLDSCRIPT := tools/buildtools/mkldscript
 MKDMADATA  := tools/buildtools/mkdmadata
-YAZ0       := tools/buildtools/yaz0
 ZAPD       := tools/ZAPD/ZAPD.out
 FADO       := tools/fado/fado.elf
-MAKEYAR    := tools/buildtools/makeyar
+MAKEYAR    := $(PYTHON) tools/buildtools/makeyar.py
+CHECKSUMMER:= $(PYTHON) tools/buildtools/checksummer.py
```

I prefer to check winter :)

---

```suggestion
CHECKSUMMER := $(PYTHON) tools/buildtools/checksummer.py
```
Could use another space. Might want to add one to the other tools as well to keep them aligned.